etna/search/tests/test_views.py

```diff
@@ -505,7 +505,17 @@ def test_render_invalid_date_range_message(self):
             self.test_url,
             data={
                 "group": "tna",
+                "created_start_date_0": "",
+                "created_start_date_1": "",
+                "created_start_date_2": "2010",
+                "created_end_date_0": "",
+                "created_end_date_1": "",
+                "created_end_date_2": "2005",
+                "opening_start_date_0": "",
+                "opening_start_date_1": "",
                 "opening_start_date_2": "2000",
+                "opening_end_date_0": "",
+                "opening_end_date_1": "",
```

Comment:
Same issue here, I would say. This dict was purposefully minimal before, and only contained the minimal data needed for the test. Now we've inflated it, it's difficult to understand what is being tested. Let's go with a separate test.

Reply:
same reason to comment on separate test to each range.

Reply:
This IS an end-to-end test (handling a request), so happy enough for this to remain as is.